### PR TITLE
Feature/fullscreen selfie cx 2345

### PR DESCRIPTION
**Pull request by stefaniacardenas**
# Problem
Selfie UI on large screens is not consistent with the liveness one

# Solution
Selfie to adopt full container layout on desktop , similar to liveness one.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [x] Have tests passed locally?
- [x] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/CameraTypes.js [hunk 1]
```diff
@@ -30,7 +30,6 @@ type CameraPureType = {
   webcamRef: React.Ref<typeof Webcam>,
   className?: string,
   trackScreen: Function,
-  isFullScreenDesktop?: boolean,
```
Comment (elhesuu) [inline src/components/Camera/CameraTypes.js]: Great

Comment (elhesuu) [reply src/components/Camera/CameraTypes.js]: Do we need to update:
```
useFullScreen: (boolean, ?boolean) => void,
```
to 
```
 useFullScreen: boolean => void,
```

Comment (stefaniacardenas) [reply src/components/Camera/CameraTypes.js]: Yes, good point!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Camera/style.css [hunk 2]
```diff
@@ -8,38 +8,30 @@
...
+.fullScreenContainer {
   position: static;
-  .uploadFallback {
-    display: none;
-  }
 }
 
-.video-overlay {
-  /* Relatively positioned so absolute children are bound to this overlay */
+.nonFullScreenContainer {
```
Comment (stephencookdev) [inline src/components/Camera/style.css]: You could also just do `:not(.fullScreenContainer)` (supported by pretty much all browsers)

Comment (stefaniacardenas) [reply src/components/Camera/style.css]: I won't be using `:not` in this instance but I've changed the class name to something more meaningful.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
just tested on IE edge on desktop. it looks like the camera is not centred - i.e. i am seeing half of my half in the oval

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by seewah** [comment]
ok everything looks good. feel free to merge!

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://406-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-406-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**